sympy/core/exprtools.py

```diff
@@ -1299,7 +1299,9 @@ def factor_nc(expr):
     """Return the factored form of ``expr`` while handling non-commutative
     expressions.
 
-    **examples**
```

Comment:
Ah, I hadn't realized.  :)  I was also mistaken about the relative frequency of `---` versus `===`.  Apparently, the former is far more common among headings immediately before code blocks, but the latter is *way* more common among headings separated by an extra newline -- which are *way* more common in general.